apps/www/components/PoliticsQuestionnaire/Overview.js

```diff
@@ -0,0 +1,294 @@
+import { css } from 'glamor'
+import { useRouter } from 'next/router'
+import { PUBLIC_BASE_URL, ASSETS_SERVER_BASE_URL } from '../../lib/constants'
+
+import Frame from '../Frame'
+import Meta from '../Frame/Meta'
+
+import {
+  ColorContextProvider,
+  colors,
+  Editorial,
+  inQuotes,
+  Interaction,
+  NarrowContainer,
+  Container,
+  Dropdown,
+  Figure,
+  FigureImage,
+  FigureCaption,
+  FigureByline,
+} from '@project-r/styleguide'
+
+import {
+  questionColor,
+  QUESTIONS,
+  QUESTIONNAIRE_BG_COLOR,
+  QUESTIONNAIRE_SQUARE_IMG_URL,
+  ILLU_SHARE,
+  ILLU_CREDIT,
+} from './config'
+
+import { QuestionLink, SubmissionLink, AnswersChart } from './shared'
+
+import {
+  AnswersGrid,
+  AnswersGridCard,
+} from '../Questionnaire/Submissions/AnswersGrid'
+
+import HeaderShare from './HeaderShare'
+
+import { ShareImageSplit } from '../Questionnaire/Submissions/ShareImageSplit'
+import { getTypeBySlug } from './utils'
+
+// filter needs to be this text/value object
+const PARTIES = [
+  { text: 'Alle', value: undefined },
+  { text: 'SVP', value: 'SVP' },
+  { text: 'SP', value: 'SP' },
+  { text: 'Grüne', value: 'GRÜNE' },
+  { text: 'Mitte', value: 'M-E' },
+  { text: 'FDP', value: 'FDP-Liberale' },
+  { text: 'LDP', value: 'LDP' },
+  { text: 'glp', value: 'glp' },
+  { text: 'EVP', value: 'EVP' },
+]
+
+export const Filters = () => {
+  const router = useRouter()
+  const query = router.query
+
+  return (
+    <NarrowContainer>
+      <div {...styles.filterContainer}>
+        <Dropdown
+          label='Partei'
+          items={PARTIES}
+          value={query.party}
+          onChange={(item) => router.push({ query: { party: item.value } })}
+        />
+      </div>
+    </NarrowContainer>
+  )
+}
+
+export const SubmissionsOverview = ({ submissionData }) => {
+  const router = useRouter()
+  const {
+    query: { image },
+  } = router
+  const urlObj = new URL(router.asPath, PUBLIC_BASE_URL)
+  const url = urlObj.toString()
+
+  const shareImageUrlObj = urlObj
+  shareImageUrlObj.searchParams.set('image', true)
+  const shareImageUrl = shareImageUrlObj.toString()
```

Comment:
I'd move everything related with the shareImageURL into a useMemo, as it doesn't really have to change on each render.